recipes/basecamp/webview.js

```diff
@@ -16,10 +16,10 @@ module.exports = Ferdium => {
   const modal = document.createElement('div');
 
   const waitFor = (condition, callback) => {
-    if (!condition()) {
-      window.setTimeout(waitFor.bind(null, condition, callback), 100);
-    } else {
```

Comment:
Code like this was autofixed by https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.0/docs/rules/no-negated-condition.md lint rule